_sass/imports/_general.scss

```diff
@@ -164,8 +164,8 @@ ol {
   padding: 0 0 0 1.33333em;
 }
 
-li > ul,
-li > ol {
+li>ul,
```

Comment:
When you saved the file, your editor also formatted it in a different standards as we are using in other `.scss` files. If you have enabled `format on save`, please disable it before saving. and thanks for the PR.

Reply:
Yeah, that's my bad. I'll get rid of auto formatting.